scripts/dockerfiles/Dockerfile.plugins-windows

```diff
@@ -53,7 +52,7 @@ RUN git clone $CLOUDWATCH_PLUGIN_CLONE_URL /cloudwatch
 WORKDIR /cloudwatch
 RUN if [ -n "$CLOUDWATCH_PLUGIN_BRANCH" ];then git fetch --all && git checkout $CLOUDWATCH_PLUGIN_BRANCH && git remote -v;fi
 RUN if [ -z "$CLOUDWATCH_PLUGIN_BRANCH" ];then git fetch --all --tags && git checkout tags/$CLOUDWATCH_PLUGIN_TAG -b $CLOUDWATCH_PLUGIN_TAG && git describe --tags;fi
-RUN go mod download
+RUN go mod download || ( go env -w GOPROXY=direct && go mod download )
```

Comment:
why parenthesis around the retry? Is this a bash thing?

Reply:
without parenthesis, we will always run `go mod download` again in logic. Here are the test case results
<img width="607" alt="Screenshot 2023-03-27 at 2 54 45 PM" src="https://user-images.githubusercontent.com/111381953/228088083-3e2500f2-8891-44e9-acec-87ce26a39f06.png">

Reply:
<img width="597" alt="Screenshot 2023-03-27 at 3 06 04 PM" src="https://user-images.githubusercontent.com/111381953/228088556-0ed48019-8222-4833-a184-
<img width="743" alt="Screenshot 2023-03-27 at 3 07 59 PM" src="https://user-images.githubusercontent.com/111381953/228088562-217779c5-e2fa-4ca7-8fa8-5790aee10944.png">
5ff8fe0568a4.png">

Reply:
<img width="597" alt="Screenshot 2023-03-27 at 3 06 04 PM" src="https://user-images.githubusercontent.com/111381953/228088656-23a0f65f-53ad-4348-a638-cf6447d54dab.png">